examples/visual-tests/amp-story/amp-story-sidebar.html

```diff
@@ -29,18 +29,19 @@
   </head>
 
   <body>
-    <amp-story standalone publisher="AMP Team" title="Visual Diff Test"
+    <amp-story id = "story" standalone publisher="AMP Team" title="Visual Diff Test"
         publisher-logo-src="https://www.ampproject.org/static/img/icons/120.png"
         poster-portrait-src="https://picsum.photos/900/1600?image=981"
         poster-landscape-src="https://picsum.photos/1600/900?image=981"
         poster-square-src="https://picsum.photos/1600/1600?image=981">
       <amp-sidebar id="sidebar1" layout="nodisplay" >
         <button id="close-button" on="tap:sidebar1.close"> close </button>
         <ul>
-          <li>Nav item 1</li>
-          <li><a href="#idTwo" on="tap:idTwo.scrollTo">Nav item 2</a></li>
-          <li>Nav item 3</li>
-          <li><a href="#idFour" on="tap:idFour.scrollTo">Nav item 4</a></li>
+          <li><span><a href="http://google.com">Out item 1</a></span></li>
+          <li><span><a href="http://localhost:8000/examples/visual-tests/amp-story/amp-story-sidebar.html#page=page-2">Absolute fragment link</a></span></li>
```

Comment:
These examples are intended to work regardless of the server.  This will break if someone changes the port, or serves their examples off of, for example, Firebase.

Reply:
Ahh, yes. Thank you for pointing this out! Done.